src/particles/plasma/PlasmaParticleContainer.cpp

```diff
@@ -386,11 +388,11 @@ IonizationModule (const int lev,
                 amrex::ParticleReal xp, yp, zp;
                 getPosition(ip, xp, yp, zp);
 
-                pstruct_elec[pidx].id()   = pid_start + pid;
-                pstruct_elec[pidx].cpu()  = lev; // current level
-                pstruct_elec[pidx].pos(0) = xp;
-                pstruct_elec[pidx].pos(1) = yp;
-                pstruct_elec[pidx].pos(2) = zp;
+                int_arrdata_elec[PlasmaIdx::id      ][pidx] = pid_start + pid;
+                int_arrdata_elec[PlasmaIdx::cpu     ][pidx] = lev; // current level
+                arrdata_elec[PlasmaIdx::x       ][pidx] = xp;
+                arrdata_elec[PlasmaIdx::y       ][pidx] = yp;
+                arrdata_elec[PlasmaIdx::z       ][pidx] = zp;
```

Comment:
These (and the lines below) could be shrunk to align with x_prev.
```suggestion
                arrdata_elec[PlasmaIdx::x    ][pidx] = xp;
                arrdata_elec[PlasmaIdx::y    ][pidx] = yp;
                arrdata_elec[PlasmaIdx::z    ][pidx] = zp;
```